src/main/java/com/lob/model/Letter.java

```diff
@@ -93,7 +94,8 @@ public Letter(
         @JsonProperty("color") final boolean color,
         @JsonProperty("double_sided") final boolean doubleSided,
         @JsonProperty("address_placement") final String addressPlacement,
-        @JsonProperty("return_envelope") final boolean returnEnvelope,
+        @JsonProperty("return_envelope") final Object returnEnvelope,

```

Comment:
Just making sure - does the `Object` type allow both strings and booleans? Since our API allows either for this param.

Reply:
If so, would `Object` be too broad?

Reply:
Sorry just now reading your pr description.

Reply:
This is why I wanted to get eyes on this. It's technically 3 types based on what I'm seeing. When we send the create either a boolean or String are value, but the value that comes back is either a boolean or an object (reference the "Areas of Concern" in the PR description)

What I'm thinking is that I need to create a class that is used in all code except when the parameter is set in the RequestBuilder

I wanted to get other people's thoughts before unilaterally made that decision (I'm not familiar with Java)

Reply:
@martinmhan I went ahead and pushed a change that implements what I was thinking

Reply:
Nice. Was looking for similar examples and saw that the `from` and `to` Letter creation API params are also either a string ID or an object, if you want to pattern it after that.

Reply:
Also, not a Java expert either, but I _think_ you can use method overloading for `setReturnEnvelope` instead of having two methods.

Reply:
Fyi feel free to ping eng-rubber-duck if you need Java expertise